includes/Core/Assets/Script.php

```diff
@@ -68,11 +68,13 @@ public function register( Context $context ) {
 		$src     = $this->args['src'];
 		$version = $this->args['version'];
 
-		$filename = Manifest::get_filename( $this->handle );
+		if ( $src ) {
```

Comment:
This added condition prevents unnecessary lookups in the manifest for scripts that don't have an `src`, particularly `Script_Data` instances.